src/test/jtx/impl/Env.cpp

```diff
@@ -59,6 +59,7 @@ namespace jtx {
 Env::AppBundle::AppBundle(beast::unit_test::suite& suite,
     std::unique_ptr<Config> config,
     std::unique_ptr<Logs> logs)
+    : AppBundle()
```

Comment:
This is the one line of code that fixes the problem.  All other changes are in support of this one line.

This line says:  Get `AppBundle` constructed in a `noexcept` manner (the default constructor is `noexcept`), and _then_ construct it some more with some things that might throw.  If any of them do, `~AppBundle()` will clean up the mess.  Without this line, `~AppBundle()` doesn't run if something throws.

Reply:
Thanks for pointing that out.  That's a really interesting use of delegating constructors.  I'll have to try to remember that.  Very cool.